react/features/toolbox/components/ToolboxItem.native.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import React from 'react';
+import { TouchableHighlight } from 'react-native';
+
+import { Icon } from '../../base/font-icons';
+
+import AbstractToolboxItem from './AbstractToolboxItem';
+import type { Props } from './AbstractToolboxItem';
+
+/**
+ * Native implementation of {@code AbstractToolboxItem}.
+ */
+export default class ToolboxItem extends AbstractToolboxItem<Props> {
+    /**
+     * Transform the given (web) icon name into a name that works with
+     * {@code Icon}.
+     *
+     * @private
+     * @returns {string}
+     */
+    _getIconName() {
+        const { iconName } = this.props;
+
+        return iconName.replace('icon-', '').split(' ')[0];
```

Comment:
While not pretty, this allows us to make icon handling platform agnostic, so I'll take it as a win :-)